src/main/java/com/google/sps/servlets/PersonServlet.java

```diff
@@ -55,11 +55,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       response.sendError(400);
       return;
     }
-    if (!authenticateRequest(personRequest)) {
-      response.sendError(HttpServletResponse.SC_UNAUTHORIZED);
-      return;
-    }
-    personDao.update(Person.create(personRequest));
+    UserService userService = UserServiceFactory.getUserService();
```

Comment:
Can this be initialized once for the whole servlet? It's probably nontrivial to construct.

Reply:
Done